Makefile

```diff
@@ -2,28 +2,28 @@ IMAGE_REPO=infracloud/botkube
 TAG=$(shell cut -d'=' -f2- .release)
 
 .DEFAULT_GOAL := build
-.PHONY: release git-tag check-git-status build pre-build tag-image publish helm-chart
+.PHONY: release git-tag check-git-status build container-image pre-build tag-image publish
 
 #Docker Tasks
 #Make a release
-release: check-git-status build tag-image publish helm-chart git-tag 
+release: check-git-status container-image tag-image publish git-tag 
 	@echo "Successfully released version $(TAG)"
 
 #Create a git tag
 git-tag:
 	@echo "Creating a git tag"
-	@git add .
-	@git commit -m "Bumped to version $(TAG)" ;
+	@git add .release helm/botkube deploy-all-in-one.yaml deploy-all-in-one-tls.yaml CHANGELOG.md
+	@git commit -m "Release $(TAG)" ;
 	@git tag $(TAG) ;
 	@git push --tags origin master;
```

Comment:
Are we pushing to master ? How does this sync back to develop ?

Reply:
Better to push to develop